quantum/nodes/classical-register/classical-register.js

```diff
@@ -47,6 +47,10 @@ module.exports = function(RED) {
       };
       flowContext.set('quantumCircuit[' + msg.payload.register.toString() + ']', register);
 
+      // get quantum circuit config and circuit ready event from flow context
+      let quantumCircuitConfig = flowContext.get('quantumCircuitConfig');
+      let circuitReady = flowContext.get('isCircuitReady');
```

Comment:
It doesn't seem like `circuitReady` is being used - can it be removed?

Reply:
Yeah it should be fine to remove it, since the classical register node does not have an output so no need for it to wait for quantum circuit to initialise.